Practice1/src/index.html

```diff
@@ -0,0 +1,214 @@
+				<p class="font-little">After having found the right doctor and the insurance company, you can
+					placea phone call to the venue and manager the booking.
+				</p>
+			</div>
+		</div>
+	</section>
+	<section class="page-read-health">
+		<img class="page-read-health-img" src="./assets/images/page-read-health.png" alt="background">
+		<h3 class="page-read-health-heading">Read Health Articles</h3>
+		<div class="page-read-health-container">
+			<div class="mask">
+				<div>
+					<img class="mask-img" src="./assets/images/mask.png" alt="mask">
+				</div>
+				<div class="page-read-health-blog">
+					<h4 class="top-medical-input">Top medical Proffessionals</h4>
+					<p class="lorem-input">Lorem ipsum dolor sit amet, an velit scaevola per. Sea eu probo
+						primis, ad vidit minimum fabellas vel. Ea ius nisl detraxit salutatus...
+					</p>
+					<div class="page-read-health-footer">
+						<p class="doctor-name">Dr.John Doe</p>
+						<a href="#" class="read-more">Read more </a>
+					</div>
+				</div>
+			</div>
+			<div class="mask">
+				<div>
+					<img class="mask-img" src="./assets/images/mask.png" alt="mask">
+				</div>
+				<div class="page-read-health-blog">
+					<h4 class="top-medical-input">Top medical Proffessionals</h4>
+					<p class="lorem-input">Lorem ipsum dolor sit amet, an velit scaevola per. Sea eu probo
+						primis, ad vidit minimum fabellas vel. Ea ius nisl detraxit salutatus...
+					</p>
+					<div class="page-read-health-footer">
+						<p class="doctor-name">Dr.John Doe</p>
+						<a href="#" class="read-more">Read more </a>
+					</div>
+				</div>
+			</div>
+			<div class="mask">
+				<div>
+					<img class="mask-img" src="./assets/images/mask.png" alt="mask">
+				</div>
+				<div class="page-read-health-blog">
+					<h4 class="top-medical-input">Top medical Proffessionals</h4>
+					<p class="lorem-input">Lorem ipsum dolor sit amet, an velit scaevola per. Sea eu probo
+						primis, ad vidit minimum
+						fabellas vel. Ea ius nisl detraxit salutatus...
+					</p>
+					<div class="page-read-health-footer">
+						<p class="doctor-name">Dr.John Doe</p>
+						<a href="#" class="read-more">Read more </a>
+					</div>
+				</div>
+			</div>
+		</div>
+	</section>
+	<section class="android">
+		<div class="android-img-component">
+			<img class="android-img" src="./assets/images/page-android.png" alt="Iphone">
+		</div>
+		<div class="android-component">
+			<h2 class="android-component-input">We are launching our App Soon!</h2>
+			<div class="common-img">
+				<img class="googleplay-icon" src="./assets/images/google-play.jpg" alt="Google Play">
+				<img class="appstore-icon" src="./assets/images/app-store.jpg" alt="App Store">
+			</div>
+			<form class="android-form">
+				<input type="email" name="email" id="mail" placeholder="Enter Your Email ID For Updates" class="android-input">
+				<button class="android-button-signup">Sign Up</button>
+			</form>
+		</div>
+	</section>
```

Comment:
Please help to reformat this file. Tab size should be 2.

Reply:
I checked and the size tab is still 2